tests/ci/create_release.py

```diff
@@ -618,10 +618,6 @@ class PackageDownloader:
         "clickhouse-server",
     )
 
-    EXTRA_PACKAGES = (
```

Comment:
Please don't remove it. Instead, fix the condition below.

We  should upload packages in existing releases.

Reply:
But they will no longer be built starting from 25.2.

Reply:
They will be built for 24.8 for another 7 months. The script should continue deploying the binaries to repositories.

The extra packages should still be pushed for releases from 24.3 until 25.1